zephyr/Kconfig.thingset

```diff
@@ -23,6 +23,19 @@ config THINGSET_ITERABLE_SECTIONS
 
           If enabled, the ThingSet context should be initialized with ts_init_global.
 
+config THINGSET_IMMUTABLE_OBJECTS
+        bool "Make data objects immutable and store them in ROM"
+        depends on THINGSET_ITERABLE_SECTIONS
+        default n
+        help
+          This option adds const specifiers to the data objects and stores them in iterable sections
+          in ROM instead of RAM.
+
+          If enabled, it's not possible to change subsets via the protocol anymore, as this
+          information is stored in the now read-only data objects array.
+
+          ToDo: Store only subset information in RAM and the rest in ROM.
+
```

Comment:
This could be clearer - The data objects themselves are not necessarily `const`, and can still reside in RAM - only the metadata i.e.  the name, access level, data type, and subsets are in ROM.

Reply:
Good point. Will improve the documentation.

Reply:
Done, please have a look again.